src/components/JdsDateInput/DateInput.stories.js

```diff
@@ -26,11 +27,9 @@ export default {
 }
 
 const Template = (args, context) => {
-  const { argTypes } = context
   return {
     name: 'JdsDateInputStories',
-    components: { JdsDateInput },
-    props: Object.keys(argTypes),
+    components: { JdsDateInput, JdsPopover },
```

Comment:
JdsPopover has been imported within JdsDateInput component. So, `import` statement can be removed.